src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -293,10 +289,11 @@ public void accountResponse_whenAccountNotExists_throwEntityNotFoundException()
         final var fakePerson1 = fakeDataGenerator.generatePerson(fakePersonId);
 
         // Build pet object
-        final var fakePet1 = fakeDataGenerator.generatePet(fakePetId);
+        final var fakePet1 = fakeDataGenerator.generateFakePet();
+        fakePet1.setId(fakePetId);
```

Comment:
Is this used anywhere in this test?

Reply:
Does not look like it to me.
Looks like it got replaced by generateDefaultTestList but never removed.